teuthology/task/install/rpm.py

```diff
@@ -139,7 +139,7 @@ def _update_package_list_and_install(ctx, remote, rpm, config):
 
     if not install_packages:
         log.info("install_packages set to False: not installing Ceph packages")
-        rpm = ["ceph-test"]
+        rpm = ["librados2", "ceph-test"]
```

Comment:
@smithfarm the comment is required for those who is reading code and it is very hard to track original patch message after many commits, so if someone decide to change the order it can introduce a bug.

Reply:
ok, comment added